Makefile

```diff
@@ -50,6 +50,16 @@ ifndef OSD_STAGING_1_OU_BASE_ID
 	$(error OSD_STAGING_1_OU_BASE_ID is undefined)
 endif
 
+.PHONY: check-sts-setup
+check-sts-setup: ## Checks if STS roles are set up correctly
+ifndef STS_JUMP_ROLE
+	$(error STS_JUMP_ROLE is undefined. STS_JUMP_ROLE is the ARN of the role we use as a bastion to access the installation role)
```

Comment:
Could potentially link to our documentation here which explains where they env variables come from:
https://github.com/openshift/aws-account-operator/blob/master/docs/1.1-InstallationPrerequisites.md#1131---jump-role